### DISCUSSION TITLE
Show and Tell: LifeSpace

### DISCUSSION THREAD

**Discussion by vishnuravi** [StanfordSpezi/.github#54] [Show and Tell]
### Description

![image](https://github.com/StanfordSpezi/.github/assets/1212163/f412a7c0-3bd1-44ec-9edb-65206ce2c41c)

To evaluate how environmental conditions influence health, we have developed a novel measure of the space within which individuals live and move. We create individual life-space maps, based on a custom smartphone application built with the Stanford Spezi framework that uses GPS-based methods. By mapping the space within which individuals live and move, we can study features of the social and built environment that support health, and identify opportunities for intervention to protect disadvantaged communities.

The LifeSpace mobile application was developed for the [Odden Research Lab](https://michelleodden.com/) in the Department of Epidemiology and Population Health at the Stanford School of Medicine.

Source: https://github.com/StanfordBDHG/LifeSpace

### Contact Information and Additional Context

[Vishnu Ravi, MD](https://github.com/vishnuravi)
[Annabel Tan, MPH](https://profiles.stanford.edu/xulin-tan)
[Sylvie Dobrota Lai, MS](https://profiles.stanford.edu/sylvie-dobrota)
[Michelle Odden, PhD](https://profiles.stanford.edu/michelle-odden)

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
Amazing; thanks for sharing @vishnuravi and get to see Spezi being used in LifeSpace!
